test/components/baseAxisTests.ts

```diff
@@ -3,6 +3,14 @@
 var assert = chai.assert;
 
 describe("BaseAxis", () => {
+  function boxesOverlap(boxA: ClientRect, boxB: ClientRect) {
```

Comment:
You can use `(<any> Plottable)._Util.DOM.boxesOverlap` here (not sure if that's cleaner...)